.travis.yml

```diff
@@ -15,12 +15,21 @@ matrix:
     - <<: *node_container
       name: 'Node - build, type-check, and end-to-end tests'
       script:
-        - yarn build --verbose
-        - yarn test:ci --testPathPattern react-server address
+        - yarn build
+        - yarn type-check
```

Comment:
`build` effectively includes a `type-check`, because part of the build is generating the TypeScript outputs